spec/integration/user_can_place_first_bid_on_an_item_spec.rb

```diff
@@ -16,4 +16,21 @@
     expect(page).to have_content("Highest Bidder: #{user.name}")
     expect(page).to have_content("Highest Bid: #{user.bids.last.price}")
   end
+
+  scenario "user cannot place the same bid twice" do
+    user = create(:user)
+    item = create(:item)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+
+    visit item_path(item)
+
```

Comment:
let's add a scope top this expectation 
-in general every assertion about page content should be scoped within the relevant html or css